DFC.Api.JobProfiles.IntegrationTests/Support/Common/CommonAction.cs

```diff
@@ -0,0 +1,129 @@
+﻿using DFC.Api.JobProfiles.Common.APISupport;
+using DFC.Api.JobProfiles.Common.AzureServiceBusSupport;
+using DFC.Api.JobProfiles.IntegrationTests.Model.JobProfile;
+using DFC.Api.JobProfiles.IntegrationTests.Support.AppSettings;
+using DFC.Api.JobProfiles.IntegrationTests.Support.CommonAction.Interface;
+using Microsoft.Extensions.Configuration;
+using Newtonsoft.Json;
+using System;
+using System.Globalization;
+using System.IO;
+using System.Linq;
+using System.Net;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace DFC.Api.JobProfiles.IntegrationTests.Support.Common
+{
+    public class CommonAction : IGeneralSupport, IJobProfileSupport
+    {
+        private static readonly Random Random = new Random();
+
+        public string RandomString(int length)
+        {
+            const string chars = "ABCDEFGHIJKLMNOPQRSTUVWXYZ";
+            return new string(Enumerable.Repeat(chars, length)
+              .Select(s => s[Random.Next(s.Length)]).ToArray());
+        }
+
+        public void InitialiseAppSettings()
+        {
+            IConfigurationRoot configuration = new ConfigurationBuilder().SetBasePath(Directory.GetCurrentDirectory()).AddJsonFile("appsettings.json", optional: true, reloadOnChange: true).Build();
+            Settings.ServiceBusConfig.ConnectionString = configuration.GetSection("ServiceBusConfig").GetSection("Endpoint").Value;
```

Comment:
Can all these be read directly into a settings type, make it neater.

Reply:
I'll look into Get<T>()

Reply:
Done.